package.json

```diff
@@ -19,7 +19,7 @@
   "author": "Denis Fäcke",
   "license": "MIT",
   "dependencies": {
-    "cookie-signature": "^1.1.0",
+    "@fastify/cookie": "^7.4.0",
```

Comment:
can `fastify-plugin` ensure the version of `@fastify/cookie` is at least 7.4? or is that out of scope for that module, and it might fail at runtime when accessing non-existent methods?

Reply:
You can require for a decorator to exist: https://github.com/fastify/fastify-plugin#dependencies

Reply:
Ah, right! should add that 👍